### ISSUE TITLE
Make a new, daemonized "listen" command

### ISSUE THREAD

**Issue by jmacdotorg** [jmacdotorg/whim#25]
Running `whim daemon` leaves you looking at a foregrounded process dumping its log messages to the terminal. That's fine for development, but not for production.

If you instead run `hypnotoad [/path/to/whim]`, then the listener automatically backgrounds itself. That's production-appropriate behavior, but what an awkward thing to type.

I would very much like the former command -- or a new command, with similar syntax -- to work like the latter.

**Comment by jmacdotorg** [comment]
Let's call the new command `whim listen`, and allow `whim daemon` to continue using the default mojo command of that name.

I experimented with making a `listen` command that used Mojo::Server::Hypnotoad directly, but that module doesn't like being used outside of the `hypnotoad` command line program, per se.

I made a trivial copy of Mojo::Server::Daemon that added a single line of `$daemon->daemonize;`, and it did the right thing -- only with no feedback, and no way to stop it (other than finding the PID yourself and issuing a `kill`).

The solution seems to be somewhere in between.
